hyper_param/neural_net.py

```diff
@@ -21,11 +21,21 @@
 IMAGE_BORDER_LENGTH = 32
 NB_CLASSES_FINE = 100
 NB_CLASSES_COARSE = 20
+DATASET_SIZE = int(os.environ.get('DATASET_SIZE', 50000))
```

Comment:
Size of x_test and x_train is different. X_test has 10k elements, x_train has 50k elements.
(10000, 32, 32, 3)
(50000, 32, 32, 3)
When DATASET_SIZE is bigger than 10k, numpy returns all data and does not give an error. 

Since rule of thump is 20/80 for test and training size, if we are going to have one number for dataset size, I would suggest using something like:
TEST_SIZE=DATASET_SIZE*0.2
TRAIN_SIZE=DATASET_SIZE*0.8

Reply:
this var was only created for local tests, to run faster code, for example i can run it on 1k samples
default is 50k, and in code I added some changes